src/gpuarray_blas_cuda_cublas.c

```diff
@@ -528,6 +763,269 @@ static int dgemv(cb_order order, cb_transpose transA, size_t M, size_t N,
   return GA_NO_ERROR;
 }
 
+static int hgemvBatch(cb_order order, cb_transpose transA,
+                      size_t M, size_t N, float alpha,
+                      gpudata **A, size_t *offA, size_t lda,
+                      gpudata **x, size_t *offX, size_t incX,
+                      float beta, gpudata **y, size_t *offY, size_t incY,
+                      size_t batchCount, int flags) {
+  return GA_DEVSUP_ERROR;
+}
+
+static int sgemvBatch(cb_order order, cb_transpose transA,
+                      size_t M, size_t N, float alpha,
+                      gpudata **A, size_t *offA, size_t lda,
+                      gpudata **x, size_t *offX, size_t incX,
+                      float beta, gpudata **y, size_t *offY, size_t incY,
+                      size_t batchCount, int flags) {
```

Comment:
When I did sgemmBatch it was a mirror to the one in cublas.

But for sgemvBatch, it doesn't act the same since I use atomics and don't cover all cases.  My idea was to leave that in the interface for now so that we can later introduce different values for the flags that would modify the behavior.  This would avoid breaking the interface again in the future.